### PR TITLE
docs: all-flash mode

### PR DESCRIPTION
**Pull request by gxglls**
可对照 http://192.168.25.87:3001/next/deploy/prerequisites 进行中文文档 review，review 结束后会交给 @zhongdeyu 进行中英翻译

### CODE REVIEW

File: docs/deploy/setup-iomesh.md [hunk 1]
```diff
@@ -84,12 +84,15 @@ spec:
 ```
 
 #### Mount Type
-
-IOMesh provides 2 mount types now:
+In `mixed-flash` deployment node, IOMesh provides 2 mount types:
 
 - `cacheWithJournal`: used for performance tier of storage pool. It **MUST** ba a partitionable block device. IOMesh will partition the device into 2 paritions: one for `journal` and another for `cache`. Either `SATA` or `NVMe` SSD is recommanded.
```
Comment (nanana-w) [inline docs/deploy/setup-iomesh.md:89]: **MUST** ba --> **MUST** be
paritions --> partitions
one for `journal` and another for `cache`. --> one for `journal` and the other for `cache`.

File: docs/deploy/setup-iomesh.md [hunk 2]
```diff
@@ -84,12 +84,15 @@ spec:
...
 - `dataStore`:  used for capacity tier of storage pool. Either `SATA` or `SAS` HDD is recommanded.
```
Comment (nanana-w) [inline docs/deploy/setup-iomesh.md:90]: recommanded --> recommended

File: docs/deploy/setup-iomesh.md [hunk 3]
```diff
@@ -84,12 +84,15 @@ spec:
...
 
+In `all-flash` deployment node, IOMesh only provides 1 mount type:
+
+- `dataStoreWithJournal`: used for capacity tier of storage pool. It **MUST** ba a partitionable block device. IOMesh will partition the device into 2 paritions: one for `journal` and another for `cache`. Either `SATA` or `NVMe` SSD is recommanded.
+
```
Comment (nanana-w) [inline docs/deploy/setup-iomesh.md:95]: **MUST** ba --> **MUST** be
2 paritions --> 2 partitions
one for `journal` and another for `cache` --> one for `journal` and the other for `dataStore`

File: docs/deploy/setup-iomesh.md [hunk 4]
```diff
@@ -101,9 +104,7 @@ Device selector is defined by:
 
 All block devices selected by device selector will be mounted to IOMesh with the corresponding mount type.
 
-#### Example
-
-Here is a full example of `chunk/deviceMap` configurations:
+#### Example of deviceMap configuration in mixed-flash deployment node
```
Comment (nanana-w) [inline docs/deploy/setup-iomesh.md:107]: node --> mode

File: docs/deploy/setup-iomesh.md [hunk 5]
```diff
@@ -133,6 +134,27 @@ spec:
     # ...
 ```
 
+#### Example of deviceMap configuration in all-flash deployment node
```
Comment (nanana-w) [inline docs/deploy/setup-iomesh.md:137]: node --> mode

File: docs/deploy/prerequisites.md [hunk 6]
```diff
@@ -5,13 +5,23 @@ sidebar_label: Prerequisites
...
-- A Kubernetes (from v1.17 to v1.21) cluster with at least 3 worker nodes
-- Each worker node needs
-  - At least one free SSD for IOMesh journal and cache
-  - At least one free HDD for IOMesh datastore
-  - A 10GbE NIC or above for IOMesh storage network
-  - At least 100GB free space at /opt
+#### 磁盘需求
+##### 缓存盘
+* 全闪架构: 无需配置
+* 混闪架构: 至少一块空闲的 SSD 磁盘，SSD 磁盘容量大于 60 GB
```
Comment (kylezh) [inline docs/deploy/prerequisites.md:14]: 架构 -> 模式

File: docs/deploy/prerequisites.md [hunk 7]
```diff
@@ -5,13 +5,23 @@ sidebar_label: Prerequisites
...
+
+##### 数据盘
+* 全闪架构: 至少一块空闲的 SSD 磁盘，SSD 磁盘容量大于 60 GB
+* 混闪架构: 至少一块空闲的 HDD 磁盘，HDD 磁盘容量大于 60 GB
```
Comment (kylezh) [inline docs/deploy/prerequisites.md:18]: ditto

File: docs/deploy/setup-iomesh.md [hunk 8]
```diff
@@ -84,11 +84,14 @@ spec:
 ```
 
 #### Mount Type
+In `mixed-flash` deployment node, IOMesh provides 2 mount types:
```
Comment (kylezh) [inline docs/deploy/setup-iomesh.md:87]: mixed -> hybrid

Comment (kylezh) [reply docs/deploy/setup-iomesh.md:87]: node -> mode

File: docs/deploy/setup-iomesh.md [hunk 9]
```diff
@@ -84,11 +84,14 @@ spec:
...
 
-IOMesh provides 2 mount types now:
+- `cacheWithJournal`: used for performance tier of storage pool. It **MUST** be a partitionable block device. IOMesh will partition the device into 2 partitions: one for `journal` and another for `cache`. Either `SATA` or `NVMe` SSD is recommended.
```
Comment (kylezh) [inline docs/deploy/setup-iomesh.md:89]: `IOMesh will partition the device into 2 partitions` -> `Two partitions will be created`

File: docs/deploy/setup-iomesh.md [hunk 10]
```diff
@@ -84,11 +84,14 @@ spec:
...
+- `dataStore`:  used for capacity tier of storage pool. Either `SATA` or `SAS` HDD is recommended.
 
-- `cacheWithJournal`: used for performance tier of storage pool. It **MUST** ba a partitionable block device. IOMesh will partition the device into 2 paritions: one for `journal` and another for `cache`. Either `SATA` or `NVMe` SSD is recommanded.
-- `dataStore`:  used for capacity tier of storage pool. Either `SATA` or `SAS` HDD is recommanded.
+In `all-flash` deployment node, IOMesh only provides 1 mount type:
```
Comment (kylezh) [inline docs/deploy/setup-iomesh.md:92]: node -> mode

File: docs/deploy/setup-iomesh.md [hunk 11]
```diff
@@ -101,9 +104,7 @@ Device selector is defined by:
 
 All block devices selected by device selector will be mounted to IOMesh with the corresponding mount type.
 
-#### Example
-
-Here is a full example of `chunk/deviceMap` configurations:
+#### Example of deviceMap configuration in mixed-flash deployment mode
```
Comment (kylezh) [inline docs/deploy/setup-iomesh.md:107]: mixed -> hybrid

File: docs/deploy/setup-iomesh.md [hunk 12]
```diff
@@ -84,11 +84,14 @@ spec:
...
 
-IOMesh provides 2 mount types now:
+- `cacheWithJournal`: used for performance tier of storage pool. It **MUST** be a partitionable block device. Two partitions will be created: one for `journal` and another for `cache`. Either `SATA` or `NVMe` SSD is recommended.
+- `dataStore`:  used for capacity tier of storage pool. Either `SATA` or `SAS` HDD is recommended.
 
-- `cacheWithJournal`: used for performance tier of storage pool. It **MUST** ba a partitionable block device. IOMesh will partition the device into 2 paritions: one for `journal` and another for `cache`. Either `SATA` or `NVMe` SSD is recommanded.
-- `dataStore`:  used for capacity tier of storage pool. Either `SATA` or `SAS` HDD is recommanded.
+In `all-flash` deployment mode, IOMesh only provides 1 mount type:
+
+- `dataStoreWithJournal`: used for capacity tier of storage pool. It **MUST** be a partitionable block device. Two partitions will be created: one for `journal` and the other for `cache`. Either `SATA` or `NVMe` SSD is recommended.
```
Comment (jeffwubj) [inline docs/deploy/setup-iomesh.md:94]: `recommended` 
It is not recommended, but MUST be either SATA or NVMe SSD?

Comment (cospotato) [inline docs/deploy/setup-iomesh.md:94]: - `dataStoreWithJournal`: used for capacity tier of storage pool. It **MUST** be a partitionable block device. Two partitions will be created: one for `journal` and the other for `dataStore`. Either `SATA` or `NVMe` SSD is recommended.

File: docs/deploy/prerequisites.md [hunk 13]
```diff
@@ -5,13 +5,23 @@ sidebar_label: Prerequisites
 ---
 
 ## Installation Requirements
+#### Number of Cluster Nodes
```
Comment (jeffwubj) [inline docs/deploy/prerequisites.md:8]: Should be `Kubernetes Cluster Requirement` ? The content contains not only requirement for nodes number

Comment (gxglls) [reply docs/deploy/prerequisites.md:8]: Number of Cluster Nodes  和 Disk Requirements 是平级的标题， Number of Cluster Nodes 内只描述节点数量

Comment (kylezh) [reply docs/deploy/prerequisites.md:8]: `Number of Cluster Nodes` 这个作为标题确实比较奇怪，我也倾向于 `Kubernetes Cluster Requirement`

File: docs/deploy/prerequisites.md [hunk 14]
```diff
@@ -5,13 +5,23 @@ sidebar_label: Prerequisites
...
-  - A 10GbE NIC or above for IOMesh storage network
-  - At least 100GB free space at /opt
+#### Disk Requirements
+##### Cache Disk
+* All-flash mode: no configuration is required.
+* Hybrid-flash mode: there is at least one free SSD on each worker node, and the SSD capacity is larger than 60 GB.
+
+##### Data Disk
+* All-flash mode: there is at least one free SSD on each worker node, and the SSD capacity is larger than 60 GB.
+* Hybrid-flash mode: there is at least one free HDD on each worker node, and the HDD capacity is larger than 60 GB.
```
Comment (kylezh) [inline docs/deploy/prerequisites.md:18]: `there is` -> `there should be`

`capacity is` -> `capacity should be`

File: docs/deploy/prerequisites.md [hunk 15]
```diff
@@ -5,13 +5,23 @@ sidebar_label: Prerequisites
...
+
+#### Network Requirement
+A NIC of 10GbE or above is required for the IOMesh storage network.
+
+#### Reserved System Space
+A space of at least 100 GB is reserved in the /opt directory of each worker node for storing the IOMesh cluster metadata.
```
Comment (kylezh) [inline docs/deploy/prerequisites.md:24]: `A space of at least 100 GB is` -> `100 GB of free disk space is`

Comment (kylezh) [reply docs/deploy/prerequisites.md:24]: `of each worker node` -> `on each worker node`

File: docs/deploy/setup-iomesh.md [hunk 16]
```diff
@@ -132,7 +133,28 @@ spec:
...
+          matchExpressions:
+          - key: iomesh.com/bd-driverType
+            operator: In
+            values:
+            - SSD
+        exclude:
+        - blockdevice-097b6628acdcd83a2fc6a5fc9c301e01
+    # ...
+```
+> **_NOTE_: The blockdevice used by IOMesh cannot have an existing filesystem, please pay attention to the `FSTYPE` field output by the command `kubectl --namespace iomesh-system -owide get blockdevice`.**
```
Comment (yuqiu123) [inline docs/deploy/setup-iomesh.md:157]: filesystem后逗号改为句号，please首字母大写Please。

File: docs/deploy/prerequisites.md [hunk 17]
```diff
@@ -5,13 +5,23 @@ sidebar_label: Prerequisites
 ---
 
 ## Installation Requirements
+#### Kubernetes Cluster Requirement
```
Comment (yuqiu123) [inline docs/deploy/prerequisites.md:8]: Requirement是否需要改为复数Requirements？

File: docs/deploy/prerequisites.md [hunk 18]
```diff
@@ -5,13 +5,23 @@ sidebar_label: Prerequisites
...
+#### Disk Requirements
+##### Cache Disk
+* All-flash mode: no configuration is required.
+* Hybrid-flash mode: there is at least one free SSD on each worker node, and the SSD capacity is larger than 60 GB.
+
+##### Data Disk
+* All-flash mode: there should be at least one free SSD on each worker node, and the SSD capacity is larger than 60 GB.
+* Hybrid-flash mode: there should be at least one free HDD on each worker node, and the HDD capacity is larger than 60 GB.
+
+#### Network Requirement
```
Comment (yuqiu123) [inline docs/deploy/prerequisites.md:20]: 同上，Requirement是否需要改为复数Requirements?

File: docs/deploy/setup-iomesh.md [hunk 19]
```diff
@@ -84,11 +84,14 @@ spec:
 ```
 
 #### Mount Type
+In `hybrid-flash` deployment mode, IOMesh provides 2 mount types:
```
Comment (yuqiu123) [inline docs/deploy/setup-iomesh.md:87]: 一般1-9数字用单词表示，看是否需要将2改成two？如果统一采用阿拉伯数字，那文档中其他1-9的数字也需要改为阿拉伯数字。

File: docs/deploy/setup-iomesh.md [hunk 20]
```diff
@@ -84,11 +84,14 @@ spec:
...
 
-IOMesh provides 2 mount types now:
```
Comment (yuqiu123) [inline docs/deploy/setup-iomesh.md:88]: 同上

File: docs/deploy/setup-iomesh.md [hunk 21]
```diff
@@ -84,11 +84,14 @@ spec:
...
+- `cacheWithJournal`: used for performance tier of storage pool. It **MUST** be a partitionable block device. Two partitions will be created: one for `journal` and another for `cache`. Either `SATA` or `NVMe` SSD is recommended.
```
Comment (yuqiu123) [inline docs/deploy/setup-iomesh.md:89]: 这里的two partitions数字用的是单词，看是否需要全文保持一致。

File: docs/deploy/setup-iomesh.md [hunk 22]
```diff
@@ -84,11 +84,14 @@ spec:
...
+- `dataStore`:  used for capacity tier of storage pool. Either `SATA` or `SAS` HDD is recommended.
 
-- `cacheWithJournal`: used for performance tier of storage pool. It **MUST** ba a partitionable block device. IOMesh will partition the device into 2 paritions: one for `journal` and another for `cache`. Either `SATA` or `NVMe` SSD is recommanded.
-- `dataStore`:  used for capacity tier of storage pool. Either `SATA` or `SAS` HDD is recommanded.
+In `all-flash` deployment mode, IOMesh only provides 1 mount type:
```
Comment (yuqiu123) [inline docs/deploy/setup-iomesh.md:92]: 这里数字采用的是阿拉伯数字，看是否需要全文保持一致。

File: docs/deploy/setup-iomesh.md [hunk 23]
```diff
@@ -101,9 +104,7 @@ Device selector is defined by:
 
 All block devices selected by device selector will be mounted to IOMesh with the corresponding mount type.
 
-#### Example
-
-Here is a full example of `chunk/deviceMap` configurations:
+#### Example of deviceMap configuration in hybrid-flash deployment mode
```
Comment (yuqiu123) [inline docs/deploy/setup-iomesh.md:107]: 这里标题是否需要和前文标题的大小写保持一致？改为：Example of DeviceMap Configuration in Hybrid-Flash Deployment Mode

File: docs/deploy/setup-iomesh.md [hunk 24]
```diff
@@ -132,7 +133,28 @@ spec:
         - blockdevice-097b6628acdcd83a2fc6a5fc9c301e01
     # ...
 ```
-    > **_NOTE_: The blockdevice used by IOMesh cannot have an existing filesystem, please pay attention to the `FSTYPE` field output by the command `kubectl --namespace iomesh-system -o wide get blockdevice`.**
+
+#### Example of deviceMap configuration in all-flash deployment mode
```
Comment (yuqiu123) [inline docs/deploy/setup-iomesh.md:137]: 同上

File: docs/deploy/setup-iomesh.md (hunk 16 above)
Comment (gxglls) [reply docs/deploy/setup-iomesh.md]: done

File: docs/deploy/prerequisites.md (hunk 17 above)
Comment (gxglls) [reply docs/deploy/prerequisites.md]: done

**Review by jeffwubj — APPROVED** [review]

File: docs/deploy/prerequisites.md [hunk 25]
```diff
@@ -5,13 +5,23 @@ sidebar_label: Prerequisites
 ---
 
 ## Installation Requirements
+#### Kubernetes Cluster Requirements
+A Kubernetes (from v1.17 to v1.21) cluster with at least 3 worker nodes
```
Comment (kylezh) [inline docs/deploy/prerequisites.md:9]: missing `.`

File: docs/deploy/prerequisites.md [hunk 26]
```diff
@@ -5,13 +5,23 @@ sidebar_label: Prerequisites
...
-- A Kubernetes (from v1.17 to v1.21) cluster with at least 3 worker nodes
-- Each worker node needs
-  - At least one free SSD for IOMesh journal and cache
-  - At least one free HDD for IOMesh datastore
-  - A 10GbE NIC or above for IOMesh storage network
-  - At least 100GB free space at /opt
+#### Disk Requirements
+##### Cache Disk
+* All-flash mode: no configuration is required.
+* Hybrid-flash mode: there is at least one free SSD on each worker node, and the SSD capacity is larger than 60 GB.
```
Comment (kylezh) [inline docs/deploy/prerequisites.md:14]:
```suggestion
* Hybrid-flash mode: there should be at least one free SSD on each worker node, while the SSD capacity is larger than 60 GB.
```

File: docs/deploy/setup-iomesh.md [hunk 27]
```diff
@@ -132,7 +133,28 @@ spec:
...
+          matchExpressions:
+          - key: iomesh.com/bd-driverType
+            operator: In
+            values:
+            - SSD
+        exclude:
+        - blockdevice-097b6628acdcd83a2fc6a5fc9c301e01
+    # ...
+```
+> **_NOTE_: The blockdevice used by IOMesh cannot have an existing filesystem. Please pay attention to the `FSTYPE` field output by the command `kubectl --namespace iomesh-system -owide get blockdevice`.**
```
Comment (kylezh) [inline docs/deploy/setup-iomesh.md:157]:
```suggestion
> **_NOTE_: The blockdevice used by IOMesh should not contain an existing filesystem. Please pay attention to the `FSTYPE` field output by the command `kubectl --namespace iomesh-system -owide get blockdevice`.**
```

@yuqiu123 Please double check.

File: docs/deploy/prerequisites.md [hunk 28]
```diff
@@ -5,13 +5,23 @@ sidebar_label: Prerequisites
...
+
+##### Data Disk
+* All-flash mode: there should be at least one free SSD on each worker node, and the SSD capacity is larger than 60 GB.
+* Hybrid-flash mode: there should be at least one free HDD on each worker node, and the HDD capacity is larger than 60 GB.
```
Comment (kylezh) [inline docs/deploy/prerequisites.md:18]:
```suggestion
* All-flash mode: there should be at least one available SSD on each worker node, and the SSD capacity should be larger than 60 GB.
* Hybrid-flash mode: there should be at least one available HDD on each worker node, and the HDD capacity should be larger than 60 GB.
```

File: docs/deploy/prerequisites.md [hunk 29]
```diff
@@ -5,13 +5,23 @@ sidebar_label: Prerequisites
...
+
+#### Network Requirements
+A NIC of 10GbE or above is required for the IOMesh storage network.
```
Comment (kylezh) [inline docs/deploy/prerequisites.md:21]:
```suggestion
Network cards of 10GbE or above are required for the IOMesh storage network.
```

File: docs/deploy/prerequisites.md [hunk 30]
```diff
@@ -5,13 +5,23 @@ sidebar_label: Prerequisites
...
+
+#### Reserved System Space
+100 GB of free disk space is required in the /opt directory on each worker node for storing the IOMesh cluster metadata.
 
 ## Setup Worker Node
```
Comment (yuqiu123) [inline docs/deploy/prerequisites.md:26]: Worker Node Setup

File: docs/deploy/setup-iomesh.md [hunk 31]
```diff
@@ -84,11 +84,14 @@ spec:
 ```
 
 #### Mount Type
+In `hybrid-flash` deployment mode, IOMesh provides two mount types:
 
-IOMesh provides 2 mount types now:
+- `cacheWithJournal`: used for performance tier of storage pool. It **MUST** be a partitionable block device. Two partitions will be created: one for `journal` and another for `cache`. Either `SATA` or `NVMe` SSD is recommended.
```
Comment (yuqiu123) [inline docs/deploy/setup-iomesh.md:89]: another 改为 the other，one and the other 表示二选一，another 表示三者或三者以上。

File: docs/deploy/setup-iomesh.md (hunk 27 above)
Comment (yuqiu123) [reply docs/deploy/setup-iomesh.md:157]: That‘s fine.

File: docs/deploy/prerequisites.md [hunk 32]
```diff
@@ -5,13 +5,23 @@ sidebar_label: Prerequisites
...
+
+##### Data Disk
+* All-flash mode: there should be at least one free SSD on each worker node, and the SSD capacity is larger than 60 GB.
+* Hybrid-flash mode: there should be at least one free HDD on each worker node, and the HDD capacity is larger than 60 GB.
+
+#### Network Requirements
+A NIC of 10GbE or above is required for the IOMesh storage network.
+
+#### Reserved System Space
+100 GB of free disk space is required in the /opt directory on each worker node for storing the IOMesh cluster metadata.
```
[the remark this answers is not in the document]
Comment (nanana-w) [reply docs/deploy/prerequisites.md:24]: 这样呢？
At least 100GB of disk space is required in the /opt directory of each worker node to store IOMesh cluster metadata.

[the remark this answers is not in the document]
Comment (yuqiu123) [reply docs/deploy/prerequisites.md:24]:
```suggestion
100 GB of free disk space is required in the /opt directory on each worker node for storing the IOMesh cluster metadata. 可以的，看看100 GB是不是要空一格~
```

**Review by kylezh — APPROVED** [review]

**Review by yuqiu123 — APPROVED** [review]

**Pull request merged**